google/data_source_dns_managed_zone_test.go

```diff
@@ -60,13 +64,23 @@ func testAccDataSourceDnsManagedZoneCheck(dsName, rsName string) resource.TestCh
 func testAccDataSourceDnsManagedZone_basic() string {
 	return fmt.Sprintf(`
 resource "google_dns_managed_zone" "foo" {
-	name		= "qa-zone-%s"
-	dns_name	= "qa.test.com."
-	description	= "QA DNS zone"
+	name		= "foo-zone-%s"
+	dns_name	= "foo.test.com."
+	description	= "Foo DNS zone"
 }
 
-data "google_dns_managed_zone" "qa" {
```

Comment:
I think we wanted to keep this a different name, to differentiate between resource "google_dns_managed_zone.foo" and data source "google_dns_managed_zone.qa" - same with below. You can name to data-foo or something like that if you're against the "qa" name.

Reply:
Ok! I can change that back